.eslintrc.json

```diff
@@ -0,0 +1,64 @@
+{
+  "root": true,
+  "env": {
+    "browser": true,
+    "es6": true,
+    "node": true,
+    "jest": true
+  },
+  "extends": ["eslint:recommended", "plugin:react/recommended", "airbnb"],
```

Comment:
In this line,

` "extends": ["eslint:recommended", "plugin:react/recommended", "airbnb"],`

change to  `"extends": [ "airbnb", "plugin:prettier/recommended", "prettier/react"],`